Anime-App/Feature/Anime/View/AnimeViewController.swift

```diff
@@ -114,13 +117,15 @@ extension AnimeViewController {
         ])
     }
     
+    
     func getAnime() {
-        
+        isLoading = true
         self.spinner.startAnimating()
-        animeViewModel.getAnime { [weak self] res in
+        animeViewModel.getAnime(page: page, offset: offset) { [weak self] res in
```

Comment:
I would create a seperate function rather than using getAnime, have another function specifically for pagination.